R/cache.R

```diff
@@ -50,17 +50,12 @@ font_cache_housing <- function() {
   )
 }
 
-# Intentionally mimics `tools::R_user_dir("thematic", "cache")` (coming in R 4.0)
-# Copyright (C) 2020 The R Core Team
-# https://github.com/wch/r-source/blob/trunk/src/library/tools/R/userdir.R
+
 thematic_cache_dir <- function() {
-  home <- normalizePath("~")
-  path <- if (.Platform$OS.type == "windows") {
-    file.path(Sys.getenv("LOCALAPPDATA"), "R", "cache")
-  } else if (Sys.info()["sysname"] == "Darwin") {
-    file.path(home, "Library", "Caches", "org.R-project.R")
+  user_dir <- if (getRversion() >= "4.0") {
+    tools::R_user_dir
```

Comment:
I think this will result in an error with `R CMD check` on old versions of R.

Reply:
That section is specifically about conditional importing inside of `NAMESPACE` -- [this code passed `R CMD check`](https://github.com/rstudio/thematic/runs/1028090562)